VMEncryption/main/EncryptionConfig.py

```diff
@@ -73,6 +73,14 @@ def clear_config(self):
                 os.rename(self.encryption_environment.encryption_config_file_path, new_name)
             else:
                 self.logger.log(msg=("the config file not exist: {0}".format(self.encryption_environment.encryption_config_file_path)), level = CommonVariables.WarningLevel)
+            if clear_parameter_file:
```

Comment:
what parameter file are we talking about? Can we add a docstring to this function and parameter?